patches/tModLoader/Terraria/ModLoader/Core/ModOrganizer.cs

```diff
@@ -126,6 +126,36 @@ private enum SearchFolders { }
 			return true;
 		}
 
+		internal static IEnumerable<ulong> DownloadWorkshopDependencies() {
```

Comment:
Can we change the method name to be 'IdentifyWorkshopDepencies'? This method does no downloading

Reply:
Yeah, old name from before I merged with the download/update UI.